doc/src/Run_options.rst

```diff
@@ -503,6 +504,36 @@ e.g. the *nfile* and *fileper* keywords.  See the
 
 ----------
 
+.. _restart2info:
+
+**-restart2info restartfile keyword ...**
+
+.. versionadded:: TBD
+
+Write out some info about the restart file and and immediately exit.  This
+is the same operation as if the following 2-line input script were
+run:
+
+.. code-block:: LAMMPS
+
+   read_restart restartfile
+   info system group compute fix keyword ...
+
+The specified restartfile name may contain the wild-card character "\*".
+The restartfile name may also contain the wild-card character "%".  The
+meaning of these characters is explained on the :doc:`read_restart
+<read_restart>` documentation.  The use of "%" means that a parallel
+restart file can be read.  Note that a filename such as file.\* may need
+to be enclosed in quotes or the "\*" character prefixed with a backslash
+("\") to avoid shell expansion of the "\*" character.
+
+The :doc:`info command <info>` is already followed by the most useful
```

Comment:
I think I know what you mean by this and the 2-line equivalent above, but it is somewhat confusing.  Maybe something like this:

Optional keywords following the restart filename can be specified to output additional details about the information contained in the restart file.   The list of allowed keywords are those which the info command uses.  Note that the info keywords group, compute, fix are automatically included.  Additional keywords like coeffs or comm can be specified.

Reply:
rewritten